cmd/setapicreds.go

```diff
@@ -86,7 +88,7 @@ func (x *SetAPICreds) Execute(args []string) error {
 		apiCfg.AllowedIPs = []string{}
 	}
 
-	if err := r.SetConfigKey("JSON-API", apiCfg); err != nil {
+	if r.SetConfigKey("JSON-API", apiCfg); err != nil {
```

Comment:
This does not look right. Which `err` did you mean to check here? 

I'm thinking the linter told you that you can't declare an `err` which has already been declared. But when you remove the `:` it tells you that `err` has not be declared inside the block. If this is the case, I've solved this by declaring the `err` at the top and using `=` throughout.